app/assets/stylesheets/blacklight.scss

```diff
@@ -1,5 +0,0 @@
-@import 'bootstrap-sprockets';
```

Comment:
Think this file was pulled over? New sufia 7.2 generator doesn't supply this file